registry/quilt_server/views.py

```diff
@@ -1348,10 +1408,18 @@ def recent_packages():
     except ValueError:
         count = 10
 
+    if ALLOW_PUBLIC_USERS:
+        user = PUBLIC
+    elif ALLOW_TEAM_USERS:
+        user = TEAM
+    else:
+        # Shouldn't really happen, but let's handle this case.
+        raise ApiException(requests.codes.forbidden, "Not allowed")
+
     results = (
         db.session.query(Package, sa.func.max(Instance.updated_at))
         .join(Package.access)
-        .filter_by(user=PUBLIC)
+        .filter_by(user=user)
```

Comment:
as written makes me nervous [about permissions escalation]. maybe it's just the kwarg name `user=`. i realize the `kwarg`  not up to the caller (it has to be a column name?). just looks funny. `user=max_visibility` would probably look better (i.e. rename the local var).